docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -526,7 +536,8 @@ def delete_project_file_version_permanently(
     Returns:
         int: the number of versions deleted
     """
-    file = qfieldcloud.core.utils.get_project_file_with_versions(project.id, filename)
+    project_id = str(project.id)
+    file = qfieldcloud.core.utils.get_project_file_with_versions(project_id, filename)
 
     if not file:
```

Comment:
why not adding a specific Exception such as 
`raise FileNotFoundError(f"File '{filename}' not found for project '{project_id}'`